cmds/core/uptime/uptime.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Get the time the machine has been up
+//
+// Synopsis:
+//     uptime
+//
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+)
+
+// This function takes in the filepath to a file containing loadavgs,it then extracts and returns the three load averages
+func loadavg(filepath string) (loadaverage string, err error) {
+	loadaverages, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return "-1", errors.New("The file proc/loadavg passed into the function loadavg() contains invalid contents or does not exist")
+	}
+	loadavg := strings.Fields(string(loadaverages))
+	loadavgstring := loadavg[0] + ", " + loadavg[1] + ", " + loadavg[2]
```

Comment:
you can also just say 
return loadavg[0] + ", " + loadavg[1] + ", " + loadavg[2]

Reply:
> you need to do a git pull --rebase

Hey Ron,I am working on individual issues per commit and pushing up for review .I will rebase once I correct the last of the issues from yesterday.So by the end of today I should be done and I will rebase everything into 1 commit.

Reply:
Done